Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Thorium Brotherhood

## Changelog:
- Added a decision to invite the Thorium Brotherhood craftsmen if Chiselgrip or Drognur are held by non-Dark Plural character.

## Developer changelog:
- Added a decision.
- Added modifiers related to the TB craftsmen and bases.
- Added an on_action regulating the appearance of base modifiers.
- Added an event related to the aforementioned decision allowing to pick an inspiration for the invited craftsman.
- Added a character template for the spawnable craftsmen.
- Added a scripted trigger checking whether holders of Drognur or Chiselgrip are Dark Plural, used in decision.

## TODO:
- [x] Check whether ruler is in Azeroth or Khaz Modan regions
- [x] Add province modifiers if Chiselgrip or Drognur (Thorium Point) are held by non-Dark Plural character.
- [x] Remove province modifiers if any of the two are held by Dark Plural faith characters.
- [x] Make the event spawn an inspired courtier and province modifier for the person picking the decision.
- [x] Event localization

## Tests:

# How to test:
- To check whether Thorium Brotherhood base modifiers appear, play as holder of either Chiselgrip or Drognur and switch their religion to non-Dark Plural. The modifier should appear on yearly pulse. To make them disappear, simply change religion back to Dark Plural.
- To invite Thorium Brotherhood craftsmen, either Chiselgrip or Drognur need to be held by a character with non-Dark Plural faith. Simply change the faith of the characters in question in console and switch to any character in de jure Azeroth or Khaz Modan empires. Give them necessary things like gold and piety in order to activate the decision. It should spawn a courtier and give a county modifier in your capital province for 10 years.
